app/assets/v2/js/cart.js

```diff
@@ -435,13 +443,16 @@ Vue.component('grants-cart', {
         vm.$set(grant, 'loading', false);
       });
     },
-    contributeWithExtension: function(grant, tenant) {
+    contributeWithExtension: function(grant, tenant, data) {
```

Comment:
from your kusama pr to avoid conflict... hehehe... but i got a lot more in other parts

```
        case 'KUSAMA':
          if (data) {
            contributeWithPolkadotExtension(grant, vm, data);
          } else {
            initPolkadotConnection(grant, vm);
          }
          break;
```